### PR TITLE
Fix data report section

### PR DESCRIPTION
**Pull request by mailtime-wing**
1. fix data report tab# do not change when slide https://github.com/measurabledatatoken/mai-website/issues/21
2. data report add autoplay and set interval for 3000ms https://github.com/measurabledatatoken/mai-website/issues/23

### CODE REVIEW

File: src/components/DataReportSection/index.js [hunk 1]
```diff
@@ -30,6 +30,8 @@ const DataReportSection = () => {
     [intl.locale]
   )
 
+  //TODO: check the bug of wrapAround
```
Comment (chochihim) [inline src/components/DataReportSection/index.js]: add the url of the issue for future reference

File: src/components/DataReportSection/index.js [hunk 2]
```diff
@@ -50,7 +52,13 @@ const DataReportSection = () => {
...
         slideIndex={selectedIndex}
         beforeSlide={() => setIsSliding(true)}
-        afterSlide={() => setIsSliding(false)}
+        afterSlide={slideIndex => {
+          setIsSliding(false)
+          setSelectedIndex(slideIndex)
+        }}
+        autoplay={true}
+        autoplayInterval={3000}
+        // wrapAround={true}
```
Comment (chochihim) [inline src/components/DataReportSection/index.js]: add the TODO next to related code

File: src/components/DataReportSection/index.js (hunk 1 above)
Comment (mailtime-wing) [reply src/components/DataReportSection/index.js]: added.

File: src/components/DataReportSection/index.js (hunk 2 above)
Comment (mailtime-wing) [reply src/components/DataReportSection/index.js]: added.

**Pull request merged**